src/Microsoft.DotNet.XHarness.Tests.Runners/IgnoreFileParser.cs

```diff
@@ -80,23 +83,26 @@ public static async Task<IEnumerable<string>> ParseContentFilesAsync(string cont
             return ignoredTests;
         }
 
-        public static async Task<IEnumerable<string>> ParseTraitsContentFileAsync(string contentDir, bool isXUnit)
+        public static async Task<IEnumerable<string>> ParseTraitsFileAsync (string filePath)
         {
             var ignoredTraits = new List<string>();
-            var ignoreFile = Path.Combine(contentDir, isXUnit ? "xunit-excludes.txt" : "nunit-excludes.txt");
-            using (var reader = new StreamReader(ignoreFile))
+            using var reader = new StreamReader(filePath);
+            string line;
+            while ((line = await reader.ReadLineAsync()) != null)
             {
-                string line;
-                while ((line = await reader.ReadLineAsync()) != null)
-                {
-                    if (string.IsNullOrEmpty(line))
-                        continue;
-                    ignoredTraits.Add(line);
-                }
+                if (string.IsNullOrEmpty(line))
```

Comment:
Should this also test for `StartsWith('#')`?

Reply:
I did not add support for comments in the traits files, we could do so @akoeplinger @EgorBo ?

Reply:
up to you, I don't think we need that, I doubt we'll ever touch that file by hands.